scheduler/templates/shift_message_to_helpers.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}{% blocktranslate with shift_title=shift.task.name.strip location=shift.facility.name.strip %}Hello {{ recipient }},
+The shift manager of the shift {{ shift_title }} at {{ location }} wants to let you know:
```

Comment:
```suggestion
{% load i18n %}{% blocktranslate with shift_title=shift.task.name.strip location=shift.facility.name.strip %}Hello {{ recipient }},

The shift manager of the shift {{ shift_title }} at {{ location }} wants to let you know:
```

For readability an empty line after greeting is more convenient.